npm/ipsm/ipsm.go

```diff
@@ -36,11 +44,20 @@ type Ipset struct {
 	referCount int
 }
 
+func (ipset *Ipset) IncReferCount() {
+	ipset.referCount++
+}
+
+func (ipset *Ipset) DecReferCount() {
```

Comment:
I think it is better not to expose this function to other packages. DecReferCount() -> decReferCount